src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -230,12 +233,12 @@ private static FermiLiteAssembly interpretAssemblyData( final ByteBuffer assembl
             final List<FermiLiteAssembly.Connection> connections = new ArrayList<>(nConnections);
             while ( nConnections-- > 0 ) {
                 int overlapLen = assemblyData.getInt();
-                final boolean isRC = overlapLen < 0;
-                overlapLen = overlapLen & Integer.MAX_VALUE; // turn off highest bit (which is now in isRC)
+                final boolean isSrc3Prime = overlapLen < 0;
+                overlapLen = overlapLen & Integer.MAX_VALUE; // turn off highest bit (which is now in isSrc3Prime)
                 int contigId = assemblyData.getInt();
-                final boolean isTargetRC = contigId < 0;
-                contigId = contigId & Integer.MAX_VALUE; // turn off highest bit (which is now in isTargetRC)
-                connections.add(new FermiLiteAssembly.Connection(contigs.get(contigId), overlapLen, isRC, isTargetRC));
+                final boolean isTarget3Prime = contigId < 0;
+                contigId = contigId & Integer.MAX_VALUE; // turn off highest bit (which is now in isTarget3Prime)
+                connections.add(new FermiLiteAssembly.Connection(contigs.get(contigId), overlapLen, !isSrc3Prime, isTarget3Prime));
```

Comment:
Add a basic unit test that fails without this fix, and passes with it.

Reply:
Hard to set up, but it's the right thing to do.